src/main/scala/com/mongodb/spark/rdd/MongoRDD.scala

```diff
@@ -141,12 +140,12 @@ class MongoRDD[D: ClassTag](
              |WARNING: Partitioning failed.
              |-----------------------------
              |
-            |Partitioning using the '${readConfig.partitioner.getClass.getSimpleName}' failed.
+             |Partitioning using the '${readConfig.partitioner.getClass.getSimpleName}' failed.
              |
-            |Please check the stacktrace to determine the cause of the failure or check the Partitioner API documentation.
+             |Please check the stacktrace to determine the cause of the failure or check the Partitioner API documentation.
              |Note: Not all partitioners are suitable for all toplogies and not all partitioners support views.%n
```

Comment:
typo in `toplogies` (missing `o`)

Reply:
Good spot - thanks @jaceklaskowski  will fix in a separate commit.